packages/teams-js/src/public/meeting.ts

```diff
@@ -245,6 +245,47 @@ export namespace meeting {
     error?: SdkError;
   }
 
+  /**
+   * Interface for mic state change
+   *
+   * @beta
+   */
+  export interface MicState {
+    /**
+     * Indicates the mute status of the mic
+     */
+    isMicMuted: boolean;
+  }
+
+  /**
+   * Reasons for the app's microphone state to change
+   */
+  enum MicStateChangeReason {
+    HostInitiated,
+    AppInitiated,
+    AppDeclinedToChange,
+    AppFailedToChange,
+  }
+
+  /**
+   * Interface for RequestAppAudioHandling properties
+   *
+   * @beta
+   */
+  export interface RequestAppAudioHandlingParams {
+    /**
+     * Indicates whether the app is requesting to start handling audio, or if
+     * it's giving audio back to the host
+     */
+    isAppHandlingAudio: boolean;
+    /**
+     * Callback for the host to tell the app to change its microphone state
+     * @param micState The microphone state for the app to use
+     * @returns A promise with the updated microphone state
+     */
+    callbackMicMuteStateChangedHandler: (micState: MicState) => Promise<MicState>;
```

Comment:
"Callback" and "Handler" are repetitive and mean essentially the same thing. If you look at other similar interfaces (e.g. AuthenticateParameters), you will see that the name of these callbacks is usually something like:
successCallback
failureCallback

I think it would be much better if this was just called `micMuteStateChangedCallback`.